form/form_collections.rst

```diff
@@ -299,6 +299,25 @@ new "tag" forms. To render it, make the following change to your template:
 
         {{ form_widget(form.tags.vars.prototype.name)|e }}
 
+.. tip::
+
+    If you need to customize the rendering of your prototype, you can use `Twig macros`_ For instance, you can do :
+
+    .. code-block:: html+twig
+
+        {% macro displayTagForm(tag) %}
```

Comment:
What about https://symfony.com/doc/current/form/form_themes.html#creating-a-form-theme-in-the-same-template-as-the-form ?

I do think that the other way is more a "best practice"

Reply:
That is arguable, but I think you are right 😂. I found the macro way cool and concise, so maybee I did get a litle too much enthousiastic about it.
Closing then ? Thx